docs/ingestion/native-batch.md

```diff
@@ -192,7 +192,8 @@ that range if there's some stray data with unexpected timestamps.
 |--------|-----------|-------|---------|
 |type|The task type, this should always be `index_parallel`.|none|yes|
 |inputFormat|[`inputFormat`](./data-formats.md#input-format) to specify how to parse input data.|none|yes|
-|appendToExisting|Creates segments as additional shards of the latest version, effectively appending to the segment set instead of replacing it. The current limitation is that you can append to any datasources regardless of their original partitioning scheme, but the appended segments should be partitioned using the `dynamic` partitionsSpec.|false|no|
+|appendToExisting|Creates segments as additional shards of the latest version, effectively appending to the segment set instead of replacing it. This means that you can append new segments to any datasource regardless of its original partitioning scheme.
```

Comment:
Did you intend to break the line here? This actually breaks the table (https://github.com/apache/druid/blob/9331331b857419c6ca45c15a1852058355019c87/docs/ingestion/native-batch.md#ioconfig). You need to add `<br/>` to break the line.